src/main/java/bio/terra/cli/app/CommandRunner.java

```diff
@@ -21,10 +20,8 @@
 public abstract class CommandRunner {
   private static final Logger logger = LoggerFactory.getLogger(CommandRunner.class);
 
-  // the Java system property that allows tests to specify a SA key file for the ADC and gcloud
-  // credentials
-  @VisibleForTesting
-  public static final String CREDENTIALS_OVERRIDE_SYSTEM_PROPERTY = "TERRA_GOOGLE_CREDENTIALS";
+  // Only unit tests set this.
+  @VisibleForTesting public static final String TEST_USER_ACCESS_TOKEN = "TERRA_GOOGLE_CREDENTIALS";
```

Comment:
This feels like a generic env name for something specific to tests.  Maybe you just don't want to rename it to keep compatibilty?

Reply:
I don't understand this comment. I changed to:

```
TEST_USER_ACCESS_TOKEN_PROPERTY = "TEST_USER_ACCESS_TOKEN";
```

Hopefully that's better.

Reply:
I just meant the "TERRA_GOOGLE_CREDENTIALS" part.